README.md

```diff
@@ -43,9 +43,8 @@ By default, this package runs using your destination and the `amazon_ads` schema
 
 ```yml
 vars:
-  amazon_ads_source:
-    amazon_ads_database: your_database_name
-    amazon_ads_schema: your_schema_name 
+  amazon_ads_database: your_database_name
+  amazon_ads_schema: your_schema_name 
```

Comment:
Let's remove the hierarchy here. It is typically easier for customers to copy/paste the global variable definition rather than the hierarchy.

Reply:
Done.